packages/rocket-batch-file-process-aws-infrastructure/README.md

```diff
@@ -7,11 +7,17 @@ Every time a new File is uploaded to the Source S3 Bucket, a new Lambda function
 This Lambda function will
 
 - Split the source file in smaller chunks. The chunk size is defined by the user as input parameter of the rocket.
-- Persist the new formed chunk in the Staging Bucket.
+- Persist the new formed chunk file in the Staging Bucket.
 - Persist a new event (containing fileUri and fileSize) for each chunk in the Event Store.
 
-**Disclaimer:** As of now the rocket can only process CSV files.
+Then, for every chunk file dropped in the Staging rocket a new Lambda function will be triggered that will:
 
+- Read each chunk file line by line and persist a new event in the Events Store for each row
+
+This event is defined in the Booster application and can be consumed from the Event Handler for any kind of processing.
+You drop your file, and you implement your logic based on an event representing a line of the file.
+
+*Disclaimer: Currently the rocket supports CSV and jsonl files.* 
```

Comment:
Can we put in the docs an example of the event "the booster way"? For example, when I used this rocket on 0.11.3 version, I defined my event like this:
```
@Event
export class S3Changed {
  public constructor(readonly id: UUID, readonly s3uri: string, readonly fileSize: string) {}

  public entityID(): UUID {
    return this.id
  }
}
```
Will it have a different format now? Do I need 2 events now that we have 2 lambdas saving events? Which one per which case?

Reply:
@juanjoman Yes I will update the doc to reflect his in a better way. 

In the previous version we were reacting to the Event Handler to the following event (defined in the rocket btw)

```
import { Event } from '@boostercloud/framework-core'
import { UUID } from '@boostercloud/framework-types'

@Event
export class FileAdded {
  public constructor(readonly fileURI: string, readonly filesize: number) {}

  public entityID(): UUID {
    return this.fileURI
  }
}
```

where `FileAdded` represents the chunk file.

In the current implementation you only need the event defined in your booster apps that represents one line of your file. See the example https://github.com/theam/process-jsonl-file

I need to explain this better overall :)